core/admin/mailu/ui/views/base.py

```diff
@@ -11,6 +11,10 @@
 def index():
     return flask.redirect(flask.url_for('.user_settings'))
 
+@ui.route('/ui/')
```

Comment:
What about removing the trailing slash?

Reply:
We can't remove it otherwise it would mean "file" and wouldn't act as a wildcard
See the section about trailing slashes on https://flask.palletsprojects.com/en/2.0.x/api/#url-route-registrations